data/links.json

```diff
@@ -79,6 +79,11 @@
         "href": "https://segfault.csh.rit.edu/",
         "icon": "material:layers"
       }
```

Comment:
You added another hash to the list though, which means you have to add a comma between your addition and the previously last item.